image/templates/helm/shared/config-templates/scanner-v4/indexer-config.yaml.tpl

```diff
@@ -21,6 +21,13 @@ indexer:
       client_encoding=UTF8
     password_file: /run/secrets/stackrox.io/secrets/password
   get_layer_timeout: 1m
+  {{- if ._rox.env.centralServices }}
+  repository_to_cpe_url: https://central.stackrox.svc/api/extensions/scannerdefinitions?type=repo2cpe
```

Comment:
@mclasmeier is something like 

```
https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?type=repo2cpe
```

possible? If so, let's switch all of these to equivalents

Reply:
yes

Reply:
> @mclasmeier is something like
> 
> ```
> https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?type=repo2cpe
> ```
> 
> possible? If so, let's switch all of these to equivalents

Is there a need for the sensor URL to do the same?

Reply:
@daynewlee yes, sensor will need the same. We can't assume the namespace will be `stackrox`